http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -210,7 +210,7 @@ object LeoPubsubMessage {
                                     appName: AppName,
                                     nodepoolId: Option[NodepoolLeoId],
                                     project: GoogleProject,
-                                    createDisk: Boolean,
+                                    createDisk: Option[DiskId],
```

Comment:
this was a bit clunky as a Boolean as it duplicated the same disk lookup for the app in front leo and back leo.